pkgs/os-specific/linux/nixos-rebuild/nixos-rebuild.sh

```diff
@@ -107,6 +108,14 @@ while [ "$#" -gt 0 ]; do
         fi
         shift 1
         ;;
+      --specialisation-name|-c)
```

Comment:
`--specialisation` seems unambiguous

Reply:
I like `--specialisation` more as well, but I wanted for it to be symmetric with `--profile-name` 👀

Reply:
No need IMO. `--profile` could possibly a path, but `--specialisation` is unambiguous